packages/langium/test/grammar/type-system/type-validator.test.ts

```diff
@@ -78,4 +78,20 @@ describe('validate params in types', () => {
         expect(diagnostics.filter(d => d.severity === DiagnosticSeverity.Error)).toHaveLength(0);
     });
 
+});
+
+describe('Inferred Types Validation', () => {
+
+    test('Should validate InferredType', async () => {
```

Comment:
Could you give more information about the test case? For example, "using inferred types in cross-reference should not throw errors"

Reply:
Good point. Going w/ `inferred type in cross-reference should not produce an error`, given that this test is verifying the absence of error diagnostics.